src/components/manage/Contents/Contents.jsx

```diff
@@ -999,33 +1016,43 @@ class Contents extends Component {
                   <Menu.Menu>
                     <Menu.Item onClick={this.upload}>
                       <Icon
-                        name="upload"
+                        name={uploadSVG}
+                        color="#007eb1"
```

Comment:
@iFlameing if you are using the same color over related elements, it's better then to do it via CSS.